contributing/how_to_contribute.rst

```diff
@@ -0,0 +1,109 @@
+.. _doc_ways_to_contribute:
+
+How to contribute
+=================
+
+The Godot Engine is free and open-source. Like any community-driven project, we rely on volunteer contributions. 
+On this page we want to showcase the various ways you as users can participate - to help you find the right starting place with the skillset you have.
+Because contrary to popular opinion, we need more than just programmers on the project!
+
+
+Fundraising
+-----------
+
+- **Donate**
+
+  We created the non-profit `Godot Foundation <https://godot.foundation/>` to be able to support the Godot Engine in both matters of finance and administration. 
+  In action, this means we can hire people to work part-time or full-time on the project. 
+  These jobs include engine development as well as related tasks like code reviews, production management, community & marketing efforts, and more.
+
+  With as little as 5 Euro per month, you can help us keep going strong. 
```

Comment:
I live in a euro country but I think this could be confusing to users who don't. Maybe I'm overthinking this :thinking:

Reply:
I believe the Fund displays in EUR but I would have to check

Reply:
```suggestion
  With as little as 5 EUR per month, you can help us keep going strong. 
```

Reply:
@winston-yallow might be worth noting for you

Reply:
For now we only support payments in Euro. In some countries stripe (the payment provider) may offer automatic conversions depending on the selected payment method, but that feels out of scope to mention here.

Reply:
I kinda assumed it was internationalized, but if it isn't, then keep it.